infra-as-code/bicep/modules/policy/assignments/lib/policy_assignments/_policyAssignmentsBicepInput.txt

```diff
@@ -1,170 +1,170 @@
-var varPolicyAssignmentDenyAppGWWithoutWAF = {
-	definitionID: '${modManagementGroups.outputs.outTopLevelMGId}/providers/Microsoft.Authorization/policyDefinitions/Deny-AppGW-Without-WAF'
+var varPolicyAssignmentDenyAppGwWithoutWaf = {

```

Comment:
Did you manually change the values in this file or re-run the powershell script that generates it?

Reply:
Manually changed via Ctrl+Shift+H against the entire repo. The script would have also been changed.